### DISCUSSION TITLE
Getting an error on start (Windows)

### DISCUSSION THREAD

**Discussion by TrickleDownClown** [vladmandic/sdnext#1584] [Q&A] [answered]
ImportError: DLL load failed while importing onnxruntime_pybind11_state: %1 is not a valid Win32 application.
Press any key to continue . . .

I did some checking but can't seem to find an answer.  I am unsure how to proceed.

**Comment by brknsoul** [comment, marked as answer]
Sounds like a corrupt download, perhaps?

Ensure: You're using Ptyhon 3.10 and not 3.11?

If yes, then try deleting the `venv` folder and running `webui` again. (this will re-create the venv folder, and download required files again).
